src/main/java/com/solutions/shop/service/CustomerService.java

```diff
@@ -0,0 +1,48 @@
+package com.solutions.shop.service;
+
+import com.solutions.shop.dao.JpaCustomerDao;
+import com.solutions.shop.dto.ResultDTO;
+import com.solutions.shop.model.Customer;
+import com.solutions.shop.model.User;
+import com.solutions.shop.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.*;
+
+@Service
+public class CustomerService {
+    @Autowired
+    private JpaCustomerDao jpaCustomerDao;
+
+    @Autowired
+    private UserRepository iUserRepository;
+    /*
+        public Optional<Customer> getCustomer(Integer id) {
+            return jpaCustomerDao.get(id);
+        }
+
+        public List<Customer> getAllUsers() {
+            return jpaCustomerDao.getAll();
+        }
+
+        public void updateUser(Customer customer) {
+            jpaCustomerDao.update(customer);
+        }
+
+        public void deleteUser(Customer customer) {
+            jpaCustomerDao.delete(customer);
+        }
+    */
+
+    /* Add Information to the personal account. */
```

Comment:
Incorrect javadocs

Reply:
/**
     * 
     * Adds info for User
     * 
     * @param userId User ID
     *  ...
     * @return result dto
     */